exporter.go

```diff
@@ -170,6 +173,17 @@ func main() {
 			// defer Shutdown to program exit
 			defer conn.Shutdown()
 
+			// Set rados_osd_op_timeout and rados_mon_op_timeout to avoid Mon
+			// and PG command hang.
+			// See https://github.com/ceph/ceph/blob/d4872ce97a2825afcb58876559cc73aaa1862c0f/src/common/legacy_config_opts.h#L1258-L1259
+			if err := conn.SetConfigOption("rados_osd_op_timeout", strconv.FormatInt(*cephRadosOSDOpTimeout, 10)); err != nil {
```

Comment:
We might need to set the config options before we issue cluster connect: https://docs.ceph.com/docs/luminous/rados/api/librados/#c.rados_conf_set

```
Pre
    rados_connect() has not been called on the cluster handle
```

Reply:
I see